docs/contributor/performance-tests/metrics/metrics.md

```diff
@@ -0,0 +1,690 @@
+
+Following graph show metric agent received metrics 
+![Metric agent accepted metrics](./assets/max-pod-agent-accepted-metrics.jpg)
+
+Following graph show metric gateway received metrics
+![Metric gateway accepted metrics](./assets/max-pod-gateway-accepted-metrics.jpg)
+
+Following graph show metric gateway exported metrics
+![Metric gateway exported metrics](./assets/max-pod-gateway-sent-metrics.jpg)
+
+Following graph show metric agent CPU utilization
+![Metric agent exported metrics](./assets/max-pod-agent-cpu.jpg)
+
+Following graph show metric agent memory utilization
+![Metric agent exported metrics](./assets/max-pod-agent-memory.jpg)
+
+Following graph show Kubernetes API server metrics
+![Metric agent exported metrics](./assets/max-pod-k8s-api.jpg)
+
+Test reached max pod count `1430` on cluster, all metrics are successfully received and exported to the metric gateway, no critical memory, cpu or network utilization observed.
+
+Kubernetes API request duration reach for `GET` operations at peak `~750ms` and for `LIST` operations `~900ms`.
+
+### Multiple pods and services scraped from multiple receivers
+
+Setup:
+
+Avalanche load generator configuration:
+- Metric Count: 500
+- Metric Series: 20
+- Number of label: 10
+
+Metric agent:
+
+Metric agent pipeline configuration `metrics/prometheus` configured with an additional receiver `prometheus/app-services` as below.
+```yaml
+apiVersion: v1
+data:
+  relay.conf: |
+    extensions:
+        health_check:
+            endpoint: ${MY_POD_IP}:13133
+    service:
+        pipelines:
+            metrics/prometheus:
+                receivers:
+                    - prometheus/app-pods
+                    - prometheus/app-services
+                processors:
+                    - memory_limiter
+                    - resource/delete-service-name
+                    - resource/insert-input-source-prometheus
+                    - batch
+                exporters:
+                    - otlp
+```
+Following service created for Avalanche load generator to enable metric scrape from second receiver `prometheus/app-services`
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "8080"
+    prometheus.io/scrape: "true"
+  labels:
+    app: avalanche-metric-load-generator
+  name: avalanche-metric-load-generator
+spec:
+  clusterIP: None
+  ports:
+  - name: http-metrics
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: avalanche-metric-load-generator
+  sessionAffinity: None
+  type: ClusterIP
+```
+
+Purpose of this test was to determine scrape behavior of metric agent and especially memory impact with multiple receiver under high load.
+
+Test metric generator started with 10 instances and metrics are scraped by the `prometheus/app-pods` receiver and `prometheus/app-services` receiver in same time independently.
+Metric generator instances were increased with `2` instance every minute to simulate a ramp-up of load.
```

Comment:
Same here, why are some elements formatted in code font? Does any of our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-code-font-for-these-items) apply?

Reply:
Reworked